it/it-tests/src/test/java/it/projectAdministration/ProjectLinksPageTest.java

```
@@ -0,0 +1,153 @@
+    ProjectLinkItem homepageLink = links.get(0);
+    ProjectLinkItem customLink = links.get(1);
+
+    homepageLink.getName().should(hasText("Home"));
+    homepageLink.getType().should(hasText("sonar.links.homepage"));
+    homepageLink.getUrl().should(hasText("http://example.com"));
+    homepageLink.getDeleteButton().shouldNot(Condition.present);
+
+    customLink.getName().should(hasText("Custom"));
+    customLink.getType().shouldNot(Condition.present);
+    customLink.getUrl().should(hasText("http://example.org/custom"));
+    customLink.getDeleteButton().shouldBe(Condition.visible);
+  }
+
+  @Test
+  public void should_create_link() {
+    ProjectLinksPage page = openPage();
+
+    page.getLinks().shouldHaveSize(2);
+
+    $("#create-project-link").click();
+    $("#create-link-name").setValue("Test");
+    $("#create-link-url").setValue("http://example.com/test");
+    $("#create-link-confirm").click();
+
+    page.getLinks().shouldHaveSize(3);
+
+    ProjectLinkItem testLink = page.getLinksAsItems().get(2);
+
+    testLink.getName().should(hasText("Test"));
+    testLink.getType().shouldNot(Condition.present);
+    testLink.getUrl().should(hasText("http://example.com/test"));
+    testLink.getDeleteButton().shouldBe(Condition.visible);
+  }
+
+  @Test
+  public void should_delete_link() {
+    ProjectLinksPage page = openPage();
+
+    page.getLinks().shouldHaveSize(2);
+
+    List<ProjectLinkItem> links = page.getLinksAsItems();
+    ProjectLinkItem customLink = links.get(1);
+
+    customLink.getDeleteButton().click();
+    $("#delete-link-confirm").click();
+
+    page.getLinks().shouldHaveSize(1);
+  }
+
+  private CreateWsResponse createCustomLink() {
+    return wsClient.projectLinks().create(new CreateWsRequest()
+      .setProjectKey("sample")
+      .setName("Custom")
+      .setUrl("http://example.org/custom"));
+  }
+
+  private void deleteLink(long id) {
+    try {
+      wsClient.projectLinks().delete(new DeleteWsRequest().setId(id));
```

I don't think you want it to fail silently. Eventually use `throw Throwables.propage(e)`. It's in each test that you want to handle it properly